### ISSUE TITLE
minerd.exe has stopped working

### ISSUE THREAD

**Issue by moneronoob77** [jwinterm/monerospelunker#5]
i can enter my monero wallet address, mining pool address, but when i click "start mining" i get "minerd.exe" has stopped working. i have closed "minerd.exe" in the task manager, but it still wont work.
where is the error code to give a solution?
how do i check hardware compatibility?
i am a beginner so i don't understand code, but i can copy and paste.
thanks.

**Comment by jwinterm** [comment]
If I had to guess it's because your CPU is not AES capable. That would be
first thing to check.

**Comment by moneronoob77** [comment]
thanks mate.
i used a program called hwinfo: https://www.hwinfo.com/download.php
it showed "AES-NI" as greyed out.
i checked with AMD tech support they confirm:
"AES is not capable with the processor."
will monerospelunker work with any p.c. which has an AES capable processor?
could you recommend a budget minning rig, for both cpu & gpu?
thanks again, jake.

**Comment by jwinterm** [comment]
Yes, it should work with any pc with aes capable processor, and even maybe
without, if you download and replace the binary with the non-AES version by
lucasjones, but it's really really not worth it to mine without an AES
capable CPU, which is why I only included that version.

**Comment by moneronoob77** [comment]
i was thinking i could replace my cpu with something AES capable,
but is AES only for X86 ? then i would be downgrading my cpu.
how can it be not worth it, with a faster processor?

motherboard: ASRock  N68C-S UCC
cpu:  AMD Athlon II X4 640
video card:  XFX Radeon HD 6670

please can you give steps to "download and replace the binary with the non-AES version by lucasjones"?
is there a cpu better than my current one, that will work with my motherboard and is AES capable. ?
are there diferent levels of AES capability?
please do you have a list of cpu's known to work with monerospulenker?
 thanks, jake.

**Comment by jwinterm** [comment]
https://github.com/lucasjones/cpuminer-multi
